mysql-test/suite/innodb/t/percona_force_encryption.test

```diff
@@ -0,0 +1,93 @@
+-- source include/have_innodb.inc
```

Comment:
If you have to re-push due to other review comments, then no space between -- and source